docs/release-notes/8.12.asciidoc

```diff
@@ -1,6 +1,28 @@
 [[release-notes-header-8.12.0]]
 == 8.12
 
+[discrete]
+[[release-notes-8.12.1]]
+=== 8.12.1
+
+[discrete]
+[[enhancements-8.12.1]]
+==== Enhancements
+* Introduces two new advanced settings for the Elastic Defend integration policy ({pull}175486[#175486]).
```

Comment:
Confirmed with @roxana-gheorghe that we don't need to release-note this for 8.12.1.